cookbooks/ruby/files/default/gemrc

```diff
@@ -0,0 +1,2 @@
+---
+gem: --no-ri --no-rdoc --no-document
```

Comment:
v6 supports Ruby 2.3+ (with the respective built-in rubygems). Do we even need `--no-ri --no-rdoc` here?

Reply:
We can remove `--no-ri --no-rdoc` then. I'll update the PR.